cardano-node/ChangeLog.md

```diff
@@ -1,6 +1,26 @@
 # Changelog for cardano-node
 
-## vNext
+## 1.36.0 -- November 2022
+
+### node changes
+
+ - Bumped cardano-ledger to 0.1
+ - Bumped plutus-core to 1.0
+ - Bumped ouroboros-network to commit 72863b0
+
+### consensus changes
+
+TODO
+
+### network changes
+
+TODO
+
+### ledger changes
+
+TODO
```

Comment:
Are these `TODO`s intended to be updated in a future PR?

Reply:
Yes, they will be filled later once we make a release